examples/tracing/kernel_cache_alloc_free.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+#
+# kernel_cache_alloc_free.py - Kernel memory (alloc-free) monitoring tool.
+#
+# Written as a basic tool example of using ePBF
+# to track of kernel cache allocation and deallocation process.
+#
+# Copyright (c) 2019 Jugurtha BELKALEM.
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 14-Jan-2019 Jugurtha BELKALEM Created this.
+
```

Comment:
Please add additional comments like:
This program shows to dump trace point kmem/kmem_cache_alloc and kmem/kmem_cache_free results using bcc python APIs. The same/similar result can be achieved though directly use kernel debugfs system.